darts/models/forecasting/regression_model.py

```diff
@@ -2048,3 +2022,40 @@ def _fit_model(
             sample_weight=sample_weight,
             **kwargs,
         )
+
+    def _format_samples(self, samples, labels=None):
```

Comment:
this can be removed since we drop HistGradientBoostingRegressor. It should probably be an abstractmethod